galley/pkg/server/server.go

```diff
@@ -57,6 +59,8 @@ type patchTable struct {
 	netListen             func(network, address string) (net.Listener, error)
 }
 
+var serverProbe *probe.Probe
```

Comment:
Move this variable into the scope of `RunServer()` on line 217.

Reply:
done